### PR TITLE
Add option to disable calling the billing API from the billing worker.

### PR DESCRIPTION
**Pull request by hodgestar**
This is an alternative to #1138.

### CODE REVIEW

**Comment by hodgestar** [comment]
Ready for review.

**Comment by justinvdm** [comment]
:+1: when Travis is happy.

File: go/vumitools/billing_worker.py [hunk 1]
```diff
@@ -171,8 +175,9 @@ def process_inbound(self, config, msg, connector_name):
         log.debug("Processing inbound: %r" % (msg,))
         msg_mdh = self.get_metadata_helper(msg)
         try:
-            yield self.create_transaction_for_inbound(msg)
-            msg_mdh.set_paid()
+            if not self.disable_billing:
+                yield self.create_transaction_for_inbound(msg)
+                msg_mdh.set_paid()
```
Comment (jerith) [inline go/vumitools/billing_worker.py:183]: Should we maybe log that we're not billing for a message because it's been disabled?

Comment (hodgestar) [inline go/vumitools/billing_worker.py:183]: Log the whole message, or just that we're not billing?

Comment (jerith) [inline go/vumitools/billing_worker.py:183]: Ideally, we want enough information that we can bill for these messages after the fact. That probably means logging the whole message.

Comment (hodgestar) [inline go/vumitools/billing_worker.py:183]: Logging added.

**Comment by hodgestar** [comment]
Ready for re-review.

**Comment by jerith** [comment]
:+1:

**Pull request merged**